client/my-sites/marketing/tools/index.tsx

```diff
@@ -15,32 +15,51 @@ import { getSelectedSiteSlug } from 'state/ui/selectors';
 import MarketingToolsFeature from './feature';
 import MarketingToolsHeader from './header';
 import { marketingSharingButtons, marketingTraffic } from 'my-sites/marketing/paths';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import { recordTracksEvent as recordTracksEventAction } from 'state/analytics/actions';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
 interface MarketingToolsProps {
```

Comment:
it looks like this was already a TypeScript file so take this comment with a grain of salt, also it's purely my own personal style…

we're already in the "marketing tools namespace" by virtue of the file path. why not just call this type `Props`? the more verbose our type names the more noisy our annotations get. further, I feel like the act of giving descriptive names to this type here is somewhat of an attempt at manual typing - that is, we're trying to say "these are marketing tools props" but the type system itself can figure this stuff out for us.

thoughts?

Reply:
Makes sense, I will update that.

Reply:
thanks. I hope my comment didn't sound too much like a heavy demand. we're all figuring this out together how we want to use it. thanks for being a part of that!

Reply:
Oh no it doesn't. I'm sorry if my comment didn't reflect it but I was totally on board with your suggestion 🙂.